index.html

```diff
@@ -9,44 +9,272 @@
+      <p class="about-par"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and<br> have a project you need coded, 
+        don’t hesitate to contact me.</p>
+      <button class="btn1">Get my resume</button>
+    </div>
+
+    <div class="all-about">
+<!-- card -->
+<div class="card about-card">
+  <img class="about-img" src="img3.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div>
+ <!-- card -->
+ <div class="card about-card">
+  <img class="about-img" src="img1.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div>
+ <!-- card -->
+ <div class="card about-card">
+  <img class="about-img" src="img2.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div> 
+
+<div class="contact-1">
+  <div class="contact-full">
+    <div class="background-img1"></div>
+    <div class="background-img2"></div>
+    <div class="contact-para">
+      <h1 class="sp9">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+    </div>
+      <div class="contact-mail">
+        <form action="https://formspree.io/f/xjvqvjew" method="POST">
+          <div class="input-none">
+            <input class="input-form input-name" maxlength="30" type="text" name="name" placeholder="Full name" required>
+          </div>
+         
+          <input class="input-form input-name" maxlength="30" type="text" name="name" placeholder="Full name" required>
+          <input class="input-form email-input" type="email" name="email" placeholder="email adress"  required>
+          <textarea class="text-name" maxlength="500" type="text" name="message"  cols="30" rows="10" placeholder="Hey i'd like to connect to.." required></textarea>
+    
+        <div class="d-btn">
+          <button class=" btn2" type="submit">Get in touch</button>
+        </div>
+        </form>
+        <div class="fin-p">
+          <p><a href="mailto:aminiroland01@gmail.com">aminiroland01@gmail.com</a></p>
+        </div>
+    
+      </div>
+    
```

Comment:
Kindly update the width of your card in the `About- me`  section so that it matches the required width you were to use as your cards are supposed to be in the same alignment vertically with that of the project section so increase them and they quite small 😄 

Your design :

![image](https://github.com/RolandAmini/portfolio-mobile/assets/20537712/1afc3a0b-e75c-4014-89bd-8ef931b67292)

Figma design :

![image](https://github.com/RolandAmini/portfolio-mobile/assets/20537712/71f7441c-2c71-43fd-9d4b-f503ac592b08)